istioctl/cmd/istioctl/metrics.go

```diff
@@ -161,8 +162,9 @@ func prometheusPods(client cache.Getter) (*v1.PodList, error) {
 	return obj.(*v1.PodList), nil
 }
 
-func buildPortForwarder(client *kubernetes.Client, config *rest.Config, podName string, port int) (*portforward.PortForwarder, <-chan struct{}, error) {
-	req := client.Post().Resource("pods").Namespace(istioNamespace).Name(podName).SubResource("portforward")
+// nolint: lll
+func buildPortForwarder(client *kubernetes.Client, config *rest.Config, podName string, ns string, port int, podPort int) (*portforward.PortForwarder, <-chan struct{}, error) {
```

Comment:
Can we move this into `istioctl/pkg` and add some testing?

Reply:
I will move it into _istioctl/pkg/kubernetes_.  There is currently no testing on that package.  I am concerned about creating testing for port-forward.  How about just the move?